Backend/routes/userRoutes.js

```
@@ -1,103 +1,116 @@
+            switch (role) {
+                case "Admin":
+                    newUser = new Admin(userData);
+                    break;
+                case "Shopkeeper":
+                    newUser = new Shopkeeper({ ...userData, shopName: req.body.shopName, shopLocation: req.body.shopLocation });
+                    break;
+                case "Deliveryman":
+                    newUser = new Deliveryman({ ...userData, vehicleType: req.body.vehicleType, assignedArea: req.body.assignedArea });
+                    break;
+                case "Farmer":
+                      const crops = Array.isArray(req.body.crops) ? req.body.crops.map(crop => crop.trim()) : [];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)